lib/Common/Core/CMakeLists.txt

```diff
@@ -21,3 +21,13 @@ add_library (Chakra.Common.Core STATIC
 
 target_include_directories (
     Chakra.Common.Core PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+
+if(NOT CC_XCODE_PROJECT)
```

Comment:
>f(NOT CC_XCODE_PROJECT) [](start = 1, length = 23)

This block is duplicated in multiple cmakelists.txt files. Is there a way to refactor common elements for cmakelists.txt files?